src/main/flight/interpolated_setpoint.c

```diff
@@ -58,9 +77,36 @@ FAST_CODE_NOINLINE float interpolatedSpApply(int axis, bool newRcFrame, ffInterp
         const float rxInterval = currentRxRefreshRate * 1e-6f;
         const float rxRate = 1.0f / rxInterval;
 
-        const float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
-        const float setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]) * pidGetDT();
+        float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
+        float setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]);
+        const float holdSteps = 2.0f;
+
+        // Glitch reduction code for identical packets
+        if ((setpointSpeed == 0) && (fabsf(rawSetpoint) < 0.98f * ffMaxRate[axis])) {
+            // identical packets, not at full deflection
+            if ((holdCount[axis] < holdSteps) && (fabsf(rawSetpoint) > 2.0f) && (bigStep[axis] == 0)){
```

Comment:
`bigStep[axis] == 0` is comparing a bool to an int.

If you call the variable `useBigStep` instead, `if (useBigStep[axis]) {` will become much easier to read and understand.

Reply:
@mikeller : the variable is 'there was big step in previous iteration', so bigStep is OK.
but `!bigStep[axis]` is obviously cleaner

Reply:
thanks - one day maybe I'll know .0001% of what you guys do :-)